src/.github/workflows/backend-ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Chariot Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        cd src
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Lint with flake8
+      run: |
+        flake8 chariot
+#     - name: Lint with pylint
+#       run: |
+#         pip install pylint
+#         pylint chariot
+#     - name: Typecheck with mypy
+#       run: |
+#         pip install mypy
+#         mypy chariot
```

Comment:
Is this an overindent? Does it affect behavior?

Reply:
It's commented out for now because the codebase is a long way from passing those lint tests. The amount of changes required is worth its own PR

Reply:
Alright, that’s fine.